### PR TITLE
IA-2387: Completeness stats infos

### PR DESCRIPTION
**Pull request by beygorghor**
Have a message telling you to select a form to be able to see the completeness results on a map

Related JIRA tickets : IA-2387

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

Create generic component to display an info tooltip nearby one input and use it

## How to test

Open completeness stats page and check that you have one infos icon nearby the form input.
Also double check if the org unit type dialog is still working, has i create a component for those two features

## Print screen / video

Screenshot 2023-09-04 at 11 09 00

### CODE REVIEW

**Review by hakifran — APPROVED** [review]
Good

File: hat/assets/js/apps/Iaso/components/InputWithInfos.tsx
Comment (quang-le) [inline hat/assets/js/apps/Iaso/components/InputWithInfos.tsx:1]: bluesquare-components?

**Review by quang-le — COMMENTED** [review]
Looks good to me, but I think the component could be moved to the library

File: hat/assets/js/apps/Iaso/components/InputWithInfos.tsx
Comment (quang-le) [reply hat/assets/js/apps/Iaso/components/InputWithInfos.tsx:1]: I created a ticket for that and will approve the PR as is

Comment (quang-le) [reply hat/assets/js/apps/Iaso/components/InputWithInfos.tsx:1]: https://bluesquare.atlassian.net/browse/IA-2412

**Pull request merged**